.rubocop_todo.yml

```diff
@@ -55,3 +55,12 @@ Style/Documentation:
 Style/FileName:
   Exclude:
     - 'lib/vagrant-service-manager.rb'
+
```

Comment:
I think there should be no more Rubocop changes required in this pull request, right? Also this seems to create a merge conflict when applying to master.